.gitignore

```diff
@@ -3,3 +3,4 @@
 app/app/
 __pycache__/
 venv/
+.vscode/
```

Comment:
* Added new line. 
* Was because when using locally with vscode (as we explicitly recommend with the dev container etc) I ended up with extra config files in `.vscode/`. Hyacinth also has this line in the `.gitignore` for that reason. Can remove if it's just me!

Reply:
In general, the .vscode folder and files should have stuff that's useful to others, which is why I asked.
Not a big deal either way.